internal/components/receivers/scraper_test.go

```diff
@@ -29,7 +29,6 @@ func TestScraperParsers(t *testing.T) {
 		defaultPort  int
 	}{
 		{"prometheus", "__prometheus", 0},
-		{"kubeletstats", "__kubeletstats", 0},
```

Comment:
this should still be a scraper right?

Reply:
I don't think so. It was removed from being a scraper in `internal/components/receivers/helpers.go` and now is treated differently.

Reply:
Reverted